presto-benchmark/pom.xml

```diff
@@ -92,6 +97,18 @@
 
         <!-- for testing -->
         <dependency>
+            <groupId>org.openjdk.jmh</groupId>
```

Comment:
I have moved it to main, and later then I have remembered why I placed it in test. Jmh and dependency checker plugin doesn play well with:

```
        <dependency>
            <groupId>org.openjdk.jmh</groupId>
            <artifactId>jmh-generator-annprocess</artifactId>
        </dependency>
```

This is required to run benchmarks, but dependency checker fails with "unused declared dependency". If it's marked as `<scope>runtime</scope>` or `<scope>test</scope>`, jmh then fails:

```
/usr/lib/jvm/java-8-oracle/bin/java -Dfile.encoding=UTF-8 -classpath /home/piotr/presto-td/presto-benchmark/target/classes:/home/piotr/presto-td/presto-spi/target/classes:/home/piotr/.m2/repository/io/airlift/slice/0.27/slice-0.27.jar:/home/piotr/.m2/repository/org/openjdk/jol/jol-core/0.2/jol-core-0.2.jar:/home/piotr/presto-td/presto-main/target/classes:/home/piotr/.m2/repository/it/unimi/dsi/fastutil/6.5.9/fastutil-6.5.9.jar:/home/piotr/.m2/repository/org/apache/commons/commons-math3/3.2/commons-math3-3.2.jar:/home/piotr/presto-td/presto-array/target/classes:/home/piotr/presto-td/presto-client/target/classes:/home/piotr/presto-td/presto-parser/target/classes:/home/piotr/.m2/repository/org/antlr/antlr4-runtime/4.6/antlr4-runtime-4.6.jar:/home/piotr/presto-td/presto-bytecode/target/classes:/home/piotr/.m2/repository/io/airlift/concurrent/0.139/concurrent-0.139.jar:/home/piotr/.m2/repository/io/airlift/node/0.139/node-0.139.jar:/home/piotr/.m2/repository/io/airlift/configuration/0.139/configuration-0.139.jar:/home/piotr/.m2/repository/cglib/cglib-nodep/2.2.2/cglib-nodep-2.2.2.jar:/home/piotr/.m2/repository/io/airlift/discovery/0.139/discovery-0.139.jar:/home/piotr/.m2/repository/io/airlift/event/0.139/event-0.139.jar:/home/piotr/.m2/repository/io/airlift/http-server/0.139/http-server-0.139.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-http/9.3.13.M0/jetty-http-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-io/9.3.13.M0/jetty-io-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/http2/http2-server/9.3.13.M0/http2-server-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/http2/http2-common/9.3.13.M0/http2-common-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/http2/http2-hpack/9.3.13.M0/http2-hpack-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-server/9.3.13.M0/jetty-server-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-util/9.3.13.M0/jetty-util-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-servlet/9.3.13.M0/jetty-servlet-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-security/9.3.13.M0/jetty-security-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-jmx/9.3.13.M0/jetty-jmx-9.3.13.M0.jar:/home/piotr/.m2/repository/ch/qos/logback/logback-core/1.0.13/logback-core-1.0.13.jar:/home/piotr/.m2/repository/io/airlift/jaxrs/0.139/jaxrs-0.139.jar:/home/piotr/.m2/repository/javax/activation/activation/1.1.1/activation-1.1.1.jar:/home/piotr/.m2/repository/org/glassfish/jersey/core/jersey-common/2.22.2/jersey-common-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/jersey/bundles/repackaged/jersey-guava/2.22.2/jersey-guava-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/hk2/hk2-api/2.4.0-b34/hk2-api-2.4.0-b34.jar:/home/piotr/.m2/repository/org/glassfish/hk2/hk2-utils/2.4.0-b34/hk2-utils-2.4.0-b34.jar:/home/piotr/.m2/repository/org/glassfish/hk2/hk2-locator/2.4.0-b34/hk2-locator-2.4.0-b34.jar:/home/piotr/.m2/repository/org/javassist/javassist/3.18.1-GA/javassist-3.18.1-GA.jar:/home/piotr/.m2/repository/org/glassfish/hk2/osgi-resource-locator/1.0.1/osgi-resource-locator-1.0.1.jar:/home/piotr/.m2/repository/org/glassfish/jersey/core/jersey-server/2.22.2/jersey-server-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/jersey/core/jersey-client/2.22.2/jersey-client-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/jersey/media/jersey-media-jaxb/2.22.2/jersey-media-jaxb-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/jersey/containers/jersey-container-servlet-core/2.22.2/jersey-container-servlet-core-2.22.2.jar:/home/piotr/.m2/repository/org/glassfish/jersey/containers/jersey-container-servlet/2.22.2/jersey-container-servlet-2.22.2.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/dataformat/jackson-dataformat-smile/2.8.1/jackson-dataformat-smile-2.8.1.jar:/home/piotr/.m2/repository/io/airlift/jmx/0.139/jmx-0.139.jar:/home/piotr/.m2/repository/io/airlift/jmx-http/0.139/jmx-http-0.139.jar:/home/piotr/.m2/repository/io/airlift/log-manager/0.139/log-manager-0.139.jar:/home/piotr/.m2/repository/org/slf4j/slf4j-jdk14/1.7.12/slf4j-jdk14-1.7.12.jar:/home/piotr/.m2/repository/org/slf4j/log4j-over-slf4j/1.7.12/log4j-over-slf4j-1.7.12.jar:/home/piotr/.m2/repository/org/slf4j/jcl-over-slf4j/1.7.12/jcl-over-slf4j-1.7.12.jar:/home/piotr/.m2/repository/io/airlift/resolver/resolver/1.3/resolver-1.3.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-spi/1.13.1/aether-spi-1.13.1.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-impl/1.13.1/aether-impl-1.13.1.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-util/1.13.1/aether-util-1.13.1.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-connector-file/1.13.1/aether-connector-file-1.13.1.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-connector-asynchttpclient/1.13.1/aether-connector-asynchttpclient-1.13.1.jar:/home/piotr/.m2/repository/com/ning/async-http-client/1.6.5/async-http-client-1.6.5.jar:/home/piotr/.m2/repository/io/netty/netty/3.7.0.Final/netty-3.7.0.Final.jar:/home/piotr/.m2/repository/org/apache/maven/maven-core/3.0.4/maven-core-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-settings/3.0.4/maven-settings-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-settings-builder/3.0.4/maven-settings-builder-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-repository-metadata/3.0.4/maven-repository-metadata-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-plugin-api/3.0.4/maven-plugin-api-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-model-builder/3.0.4/maven-model-builder-3.0.4.jar:/home/piotr/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.14/plexus-interpolation-1.14.jar:/home/piotr/.m2/repository/org/codehaus/plexus/plexus-utils/2.0.6/plexus-utils-2.0.6.jar:/home/piotr/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar:/home/piotr/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar:/home/piotr/.m2/repository/org/apache/maven/maven-model/3.0.4/maven-model-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-artifact/3.0.4/maven-artifact-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-aether-provider/3.0.4/maven-aether-provider-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-embedder/3.0.4/maven-embedder-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/maven-compat/3.0.4/maven-compat-3.0.4.jar:/home/piotr/.m2/repository/org/apache/maven/wagon/wagon-provider-api/2.2/wagon-provider-api-2.2.jar:/home/piotr/.m2/repository/org/sonatype/plexus/plexus-cipher/1.7/plexus-cipher-1.7.jar:/home/piotr/.m2/repository/org/codehaus/plexus/plexus-container-default/1.5.5/plexus-container-default-1.5.5.jar:/home/piotr/.m2/repository/org/apache/xbean/xbean-reflect/3.4/xbean-reflect-3.4.jar:/home/piotr/.m2/repository/org/codehaus/plexus/plexus-classworlds/2.4/plexus-classworlds-2.4.jar:/home/piotr/.m2/repository/org/slf4j/slf4j-api/1.7.12/slf4j-api-1.7.12.jar:/home/piotr/.m2/repository/io/airlift/trace-token/0.139/trace-token-0.139.jar:/home/piotr/.m2/repository/io/airlift/joni/2.1.5.1/joni-2.1.5.1.jar:/home/piotr/.m2/repository/com/teradata/re2j-td/1.4/re2j-td-1.4.jar:/home/piotr/.m2/repository/io/airlift/discovery/discovery-server/1.27/discovery-server-1.27.jar:/home/piotr/.m2/repository/io/airlift/jmx-http-rpc/0.136/jmx-http-rpc-0.136.jar:/home/piotr/.m2/repository/org/iq80/leveldb/leveldb-api/0.8/leveldb-api-0.8.jar:/home/piotr/.m2/repository/org/iq80/leveldb/leveldb/0.8/leveldb-0.8.jar:/home/piotr/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar:/home/piotr/.m2/repository/javax/annotation/javax.annotation-api/1.2/javax.annotation-api-1.2.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.8.1/jackson-databind-2.8.1.jar:/home/piotr/.m2/repository/joda-time/joda-time/2.8.2/joda-time-2.8.2.jar:/home/piotr/.m2/repository/org/weakref/jmxutils/1.19/jmxutils-1.19.jar:/home/piotr/.m2/repository/javax/validation/validation-api/1.1.0.Final/validation-api-1.1.0.Final.jar:/home/piotr/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar:/home/piotr/.m2/repository/com/google/inject/guice/4.0/guice-4.0.jar:/home/piotr/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar:/home/piotr/.m2/repository/com/google/inject/extensions/guice-multibindings/4.0/guice-multibindings-4.0.jar:/home/piotr/.m2/repository/io/airlift/http-client/0.139/http-client-0.139.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-client/9.3.13.M0/jetty-client-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/http2/http2-client/9.3.13.M0/http2-client-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/jetty-alpn-client/9.3.13.M0/jetty-alpn-client-9.3.13.M0.jar:/home/piotr/.m2/repository/org/eclipse/jetty/http2/http2-http-client-transport/9.3.13.M0/http2-http-client-transport-9.3.13.M0.jar:/home/piotr/.m2/repository/org/sonatype/aether/aether-api/1.13.1/aether-api-1.13.1.jar:/home/piotr/.m2/repository/org/ow2/asm/asm-all/5.0.4/asm-all-5.0.4.jar:/home/piotr/.m2/repository/org/jgrapht/jgrapht-core/0.9.0/jgrapht-core-0.9.0.jar:/home/piotr/.m2/repository/org/apache/bval/bval-jsr/1.1.1/bval-jsr-1.1.1.jar:/home/piotr/.m2/repository/org/apache/bval/bval-core/1.1.1/bval-core-1.1.1.jar:/home/piotr/.m2/repository/commons-beanutils/commons-beanutils-core/1.8.3/commons-beanutils-core-1.8.3.jar:/home/piotr/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar:/home/piotr/.m2/repository/javax/xml/bind/jaxb-api/2.2.6/jaxb-api-2.2.6.jar:/home/piotr/.m2/repository/com/sun/xml/bind/jaxb-impl/2.2.6/jaxb-impl-2.2.6.jar:/home/piotr/presto-td/presto-tpch/target/classes:/home/piotr/.m2/repository/io/airlift/tpch/tpch/0.8/tpch-0.8.jar:/home/piotr/presto-td/presto-memory/target/classes:/home/piotr/.m2/repository/com/google/code/findbugs/annotations/2.0.3/annotations-2.0.3.jar:/home/piotr/.m2/repository/io/airlift/json/0.139/json-0.139.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.8.1/jackson-datatype-jdk8-2.8.1.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.8.1/jackson-datatype-jsr310-2.8.1.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-guava/2.8.1/jackson-datatype-guava-2.8.1.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-joda/2.8.1/jackson-datatype-joda-2.8.1.jar:/home/piotr/.m2/repository/io/airlift/log/0.139/log-0.139.jar:/home/piotr/.m2/repository/io/airlift/units/1.0/units-1.0.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.8.1/jackson-annotations-2.8.1.jar:/home/piotr/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.8.1/jackson-core-2.8.1.jar:/home/piotr/.m2/repository/com/google/guava/guava/18.0/guava-18.0.jar:/home/piotr/.m2/repository/org/openjdk/jmh/jmh-core/1.15/jmh-core-1.15.jar:/home/piotr/.m2/repository/net/sf/jopt-simple/jopt-simple/4.6/jopt-simple-4.6.jar:/home/piotr/.m2/repository/io/airlift/stats/0.139/stats-0.139.jar:/home/piotr/.m2/repository/org/hdrhistogram/HdrHistogram/2.1.9/HdrHistogram-2.1.9.jar:/home/piotr/.m2/repository/io/airlift/bootstrap/0.139/bootstrap-0.139.jar:/home/piotr/.m2/repository/javax/ws/rs/javax.ws.rs-api/2.0.1/javax.ws.rs-api-2.0.1.jar:/home/piotr/.m2/repository/org/openjdk/jmh/jmh-generator-annprocess/1.15/jmh-generator-annprocess-1.15.jar org.openjdk.jmh.Main com.facebook.presto.benchmark.BenchmarkDecimalAggregation.benchmarkBuildHash
Exception in thread "main" java.lang.RuntimeException: ERROR: Unable to find the resource: /META-INF/BenchmarkList
	at org.openjdk.jmh.runner.AbstractResourceReader.getReaders(AbstractResourceReader.java:97)
	at org.openjdk.jmh.runner.BenchmarkList.find(BenchmarkList.java:103)
	at org.openjdk.jmh.runner.Runner.internalRun(Runner.java:256)
	at org.openjdk.jmh.runner.Runner.run(Runner.java:206)
	at org.openjdk.jmh.Main.main(Main.java:71)
```

I'm leaving it in test for now.